File: doc/snippets/rcs/revoke/snippet.js
const url = 'https://api.rcsbox.net/rcs/1.0/json/<AGENTID>/revoke>';
const headers = {
    'Authorization': 'App <APIKEY>',
    'Content-Type': 'application/json'
};
const data = {
    recipients: ["+336XXXXXXXX"],
    reference: "XXXXXXXXXXXX"
};

fetch(url, {
    method: 'POST',
    headers: headers,
    body: JSON.stringify(data)
})
    .then(response => response.json())
    .then(data => console.log(data))
    .catch(error => console.error('Error:', error));

File: doc/snippets/rcs/revoke/snippet.py
import requests
import json

url = 'https://api.rcsbox.net/rcs/1.0/json/<AGENTID>/revoke'
headers = {
    'Authorization': 'App <APIKEY>',
    'Content-Type': 'application/json'
}
data = {
    "recipients": ["+336XXXXXXXX"],
    "reference": "XXXXXXXXXXXX"
}

response = requests.post(url, headers=headers, data=json.dumps(data))

print(response.status_code)
print(response.json())
